.github/workflows/pr-differences-mutants.yml

```
@@ -38,23 +41,27 @@ jobs:
 
     needs: check-big-packages-and-shards
 
-    if: ${{ needs.check-big-packages-and-shards.outputs.run_small_packages == 'true' && needs.check-big-packages-and-shards.outputs.small_packages_with_shards == 'false' }}
+    if: |
+      ${{ needs.check-big-packages-and-shards.outputs.run_small_packages == 'true' &&
+      needs.check-big-packages-and-shards.outputs.small_packages_with_shards == 'false' }}
```

one of the benefits of moving to multiline conditionals is that you can remove the `${{ }}` chars, ex:
```suggestion
    if: |
      needs.check-big-packages-and-shards.outputs.run_small_packages == 'true' &&
      needs.check-big-packages-and-shards.outputs.small_packages_with_shards == 'false'
```